pkg/types/config.go

```diff
@@ -13,7 +13,7 @@
  *******************************************************************************/
 
 // Package vault defines structs that will be used frequently by clients which utilize HTTP transport.
```

Comment:
update the out-of-date package comment

Reply:
Remove package comment. Never thought they are usefull.